tests/test_odl.py

```diff
@@ -96,8 +96,15 @@ def test_get_license_status_document_success(self):
         # Loans expire in 21 days by default.
         now = utc_now()
         after_expiration = now + datetime.timedelta(days=23)
-        expires = urllib.parse.unquote_plus(params.get("expires")[0])
+        expires = urllib.parse.unquote(params.get("expires")[0])
+
+        # The expiration time passed to the server is associated with
+        # the UTC time zone.
+        assert expires.endswith('+00:00')
         expires = dateutil.parser.parse(expires)
```

Comment:
This wasn't catching the error because dateutil.parser.parse doesn't have a problem parsing the technically nonsensical "+00:00Z" syntax; it makes a relatively obvious guess. But the actual DPLA Exchange server refuses to make that guess.